packages/datagateway-dataview/public/res/default.json

```diff
@@ -35,6 +36,13 @@
     "end_date": "End Date",
     "start_date": "Start Date"
   },
+  "studies": {
+    "name": "Name",
+    "rb_number":"RB Number",
```

Comment:
I'd prefer this to match the ICAT field name - rb_number isn't a field - it's generally the `name` field